pulpcore/cli/core/repository.py

```diff
@@ -8,8 +10,11 @@
     name_filter_options,
     pass_pulp_context,
     pulp_group,
+    pulp_option,
+    resource_option,
     version_command,
 )
+from pulpcore.cli.core.export import _version_list_callback
```

Comment:
That is not looking like a proper import.
I think the callback belongs in core.generics.

Reply:
@mdellweg Fixing it right now, just want to say that we went export -> common.generic -> export -> core.generic so far